sites/all/modules/publizon/includes/publizon.loan.inc

```diff
@@ -98,9 +98,22 @@ function publizon_loan_create($account, $local_id) {
     // This will throw exception if not logged in.
     $creds = ding_user_get_creds();
 
+    $institution_id = NULL;
+    if (function_exists('ding_unilogin_get_user_data')) {
+      $unilogin_data = ding_unilogin_get_user_data();
+      if (isset($unilogin_data['institutions'])
+        && is_array($unilogin_data['institutions'])) {
+        $institutions = $unilogin_data['institutions'];
+        $institution = reset($institutions);
```

Comment:
Is it correct to always take the first one?

Reply:
Would it be more correct to do the test first `if (is_array('$institutions'))` and then `$institution = reset($institutions);`

Reply:
This is checked by `is_array($unilogin_data['institutions'])`. The `$institutions` variable is only introduced to be able to call `reset` which takes an array reference as argument.

Taking the first institution is the right way to do it. (It matches https://github.com/eReolen/base/blob/develop/sites/all/modules/reol_statistics/reol_statistics.module#L53-L54 which may be obsoleted by sending the institution id to Publizon.)